viYarn/Episode/view/EpisodeViewController.swift

```diff
@@ -32,23 +43,25 @@ class EpisodeTableViewController: UITableViewController {
         title.font = UIFont.systemFont(ofSize: 19, weight: .medium)
         title.textAlignment = .center
         title.textColor = .white
+        
   
         let btnDone = UIBarButtonItem(title: "Done", style: .done, target: self, action: #selector(onClickDone))
         btnDone.tintColor = .white
         navigationItem.rightBarButtonItem = btnDone
+        
+        self.view.addSubview(tableView)
```

Comment:
p1: code style: the is no need to specify self here. We can just write view.addSubview(). I think it is more clear.